components/utilities/chartLayout.js

```diff
@@ -159,7 +159,7 @@
             var plotAreaClipId = 'fcPlotAreaClip_' + (element.id || nextId());
 
             // Clipping path
-            var clippingPath = defs.selectAll('#' + plotAreaClipId).data(dummyData);
+            var clippingPath = chartElements.defs.selectAll('#' + plotAreaClipId).data(dummyData);
```

Comment:
For the clipping path, as the ID should only be set once, when the element is initially appended, I think it would be safe to chain this way (and it should be more efficient too!):

```js
var clippingPath = chartElements.defs.selectAll('#' + plotAreaClipId).data(dummyData);
clippingPath.enter().append('clipPath')
    .attr('id', plotAreaClipId);
clippingPath.exit().remove();
```